docs/manual/linux-udev.md

```diff
@@ -3,19 +3,93 @@ layout: default
 title: Configuring udev on Linux
 parent: User manual
 nav_order: 6
+has_toc: true
 redirect_from:
   - /getting-started/linux-udev.html
 ---
 
-# Configuring udev on Linux
+# Configuring `udev` rules for VIA and Vial on Linux
 
-In order for your keyboard to be detected on Linux, you need to set up a custom udev rule. Open a terminal and execute the following commands to set this up:
+In order for your keyboard to be detected by both VIA and Vial on Linux, you need to set up a custom `udev` rule. Both use the `hidraw` linux driver to directly communicate configurations to your keyboard. In most situations it should be safe to use a generalized `udev` rule for all `hidraw` devices, but in other cases where you prefer or are required to restrict access on a device basis, it is possible to create a `udev` rule specifically for your keyboard.
+
+The following guides will show you how to implement these `udev` rules. You will require root access to create the files necessary.
+
+## Vial `udev` rule
+
+For a generalized access rule for any device with Vial firmware, write this text to `/etc/udev/rules.d/99-vial.rules`:
+
+```
+KERNEL=="hidraw*", SUBSYSTEM=="hidraw", ATTRS{serial}=="vial:f64c2b3c", MODE="0660", GROUP="users", TAG+="uaccess", TAG+="udev-acl"
+```
+
+**OR** run this in your shell (replace `sudo` with `pkexec` or `doas` if needed):
+
+```
+sudo echo "KERNEL==\"hidraw*\", SUBSYSTEM==\"hidraw\", ATTRS{serial}==\"vial:f64c2b3c\", MODE=\"0660\", GROUP=\"users\", TAG+=\"uaccess\", TAG+=\"udev-acl\"" > /etc/udev/rules.d/99-vial.rules
```

Comment:
pretty sure this doesn't actually work since the `>` redirect is handled by your shell, not but sudo. you need `sudo tee` similar to how it was set up previously

Reply:
Ah, yeah that's probably true. I typically don't use `sudo` in commands like that so it must not have been obvious to me. Thanks for pointing that out.

Reply:
I just modified the one-liners to automatically place the currently logged in user's main group ID in the `GROUP` field. I also specifically stated in the examples that the `users` group can be replaced if it doesn't exist on ones system.